magento/pwa-studio
# [BUGFIX] The browser back button wasn't responding

## Description

Check for [#3077](https://github.com/magento/pwa-studio/issues/3077) for explanation.

## Related Issue





Closes #3077.

## Acceptance



### Verification Stakeholders

### Specification



### Verification Steps


Go to https://venia.magento.com/default/venia-tops/venia-blouses.html?page=1
Add some filters
Now click on the browser back button once
You should see we're back to the prev page/state

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
